### PR TITLE
Collector Dockerfile, Package Updates, Prettier

### PR DESCRIPTION
**Pull request by zemberdotnet**

### CODE REVIEW

File: .github/workflows/package-publish.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+name: Build and Package
```
Comment (VDubber) [inline .github/workflows/package-publish.yml:1]: We need to consider moving our GH workflow files into a repo and referencing them. That way we drive down maintenance. I'll try to find more info

Comment (VDubber) [reply .github/workflows/package-publish.yml:1]: I believe we can add them [here](https://github.com/JupiterOne/.github/tree/main/.github/workflows)
And then reference them like [this](https://github.com/JupiterOne/web-integrations/blob/main/.github/workflows/deploy.yml)

Comment (VDubber) [reply .github/workflows/package-publish.yml:1]: Please consider doing this work for this PR.

Comment (maxheckel) [reply .github/workflows/package-publish.yml:1]: Didn't know this was possible, dope!

**Review by maxheckel — APPROVED** [review]

**Pull request merged**